pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "manim"
+version = "0.2.0"
+description = "Animation engine for explanatory math videos."
+authors = ["3b1b <grant@3blue1brown.com>"]
+license="MIT"
+readme="README.md"
+repository="https://github.com/manimcommunity/manim"
+documentation="https://eulertour.com"
```

Comment:
Issue: please change to https://manimce.readthedocs.io/en/latest/

Reply:
(in the future we will have to change to en/stable/, but let's link to en/latest/ for now)